### DISCUSSION TITLE
windows 11 taskbar style automatically is shifted to the top upon appearing

### DISCUSSION THREAD

**Discussion by Ticha22608** [valinet/ExplorerPatcher#2077] [Q&A] [answered]
whenever i try to set my taskbar to windows 11 style, it's automatically shifted to the top for some reason. i can't change it from explorerpatcher as the option's not there, and it only happens in this user account. in my other user account it works as intended.

i can use the windows 10 style and have my taskbar at the bottom, just not the windows 11 one. and this only happens on my main display: 
![image](https://github.com/valinet/ExplorerPatcher/assets/40746497/72fb94a5-2359-45ba-9ae1-aced076df903)

**Comment by pyrates999** [comment, marked as answer]
Do you have any other software that modifies the GUI installed beside EP?

**Reply by Ticha22608** [reply]
i'm such an idiot lol .... i never checked this because i couldn't find anything in my other UI modifying program (start11) because the option doesn't show up unless i'm using windows 11 style and it hadn't done this in my other user account ... thank you for making me check that
